packages/components/MenuButton/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "@fluentui-react-native/menu-button",
+  "version": "0.0.0",
```

Comment:
I would recommend starting at 0.1.0 for the version. Not sure if 0.0.0 has special behavior

Reply:
I thought I updated it. Will fix, thank you